gui/halt.php

```diff
@@ -107,11 +107,15 @@ function check_stat($r) {
 		echo "<tr>";
 		echo "<td align='center'>".$element['iteration']."</td>";
 
-		if ($element['status'] > 1000){
-			$sum += 1;
-			$statuses['in progress'] += 1;
-			echo "<td>".($element['status']-1000)."%</td><td>Job in progress</td></tr>"; // Add a line break after each inner array
-		}else{
+        if ($element['status'] > 1000 & $element['status']<1100 ){
+            $sum += 1;
+            $statuses['in progress'] += 1;
+            echo "<td>".($element['status']-1000)."%</td><td>Job in progress</td></tr>"; // Add a line break after each inner array
+        }
+        elseif($element['status'] == 1100){
+                echo "<td>".($element['status']-1000)."%</td><td>Job finished</td></tr>";
```

Comment:
Job of status 100% is still in progress I suppose - meta file is saved and results are exported to DB. Statuses 0-99 stand for finished eventually.
I suggest to restore those changes - to be discussed. Unless there are bugs in halt.php regarding issue at hand?

Reply:
Understandable. 
No, everything works well.